.github/workflows/desktop-build-deploy.yml

```
@@ -125,12 +184,16 @@
           EOF
 
           # Upload each file with appropriate content-type and cache-control
-          for file in dist/artifacts/*; do
+          for file in dist/artifacts/all/*; do
             filename=$(basename "$file")
             content_type="application/octet-stream"
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)